src/debug.c

```diff
@@ -2127,6 +2382,59 @@ void BufferExpansionVersion(struct ScriptContext *ctx)
         string = StringCopy(string, sText_Unreleased);
 }
 
+static void DebugAction_TimeMenu_PrintTime(u8 taskId)
+{
+    Debug_DestroyMenu_Full(taskId);
+    LockPlayerFieldControls();
+    Debug_DestroyMenu_Full_Script(taskId, Debug_EventScript_TellTheTime);
+}
+
+void DebugMenu_CalculateTime(struct ScriptContext *ctx)
+{
+    if(OW_USE_FAKE_RTC == TRUE) {
+        struct SiiRtcInfo *rtc = FakeRtc_GetCurrentTime();
+        StringExpandPlaceholders(gStringVar1, gDayNameStringsTable[rtc->dayOfWeek]);
+        ConvertIntToDecimalStringN(gStringVar2, rtc->hour, STR_CONV_MODE_LEFT_ALIGN, 3);
+        ConvertIntToDecimalStringN(gStringVar3, rtc->minute, STR_CONV_MODE_LEADING_ZEROS, 2);
+    }
+    else {
+        u32 day = ((gLocalTime.days - 1) + 6) % 7 ;
+        RtcCalcLocalTime();
+        StringExpandPlaceholders(gStringVar1, gDayNameStringsTable[day]);
+        ConvertIntToDecimalStringN(gStringVar2, gLocalTime.hours, STR_CONV_MODE_LEFT_ALIGN, 3);
+        ConvertIntToDecimalStringN(gStringVar3, gLocalTime.minutes, STR_CONV_MODE_LEADING_ZEROS, 2);
+    }
+
+}
+
+static void DebugAction_TimeMenu_PrintTimeOfDay(u8 taskId)
+{
+    Debug_DestroyMenu_Full(taskId);
+    LockPlayerFieldControls();
+    Debug_DestroyMenu_Full_Script(taskId, Debug_EventScript_PrintTimeOfDay);
+}
+
+void DebugMenu_CalculateTimeOfDay(struct ScriptContext *ctx)
+{
+    switch(GetTimeOfDay())
+    {
+        case TIME_MORNING:
+            StringExpandPlaceholders(gStringVar1, sDebugText_TimeMenu_Morning);
+            break;
+        case TIME_DAY:
+            StringExpandPlaceholders(gStringVar1, sDebugText_TimeMenu_Day);
+            break;
+        case TIME_EVENING:
+            StringExpandPlaceholders(gStringVar1, sDebugText_TimeMenu_Evening);
+            break;
+        case TIME_NIGHT:
+            StringExpandPlaceholders(gStringVar1, sDebugText_TimeMenu_Night);
+            break;
+        default: //Added due to TIMES_OF_DAY_COUNT causing errors.
```

Comment:
```suggestion
        default:
```
Comment is not necessary